rs/nervous_system/tools/release-runscript/src/utils.rs

```diff
@@ -92,3 +92,134 @@ pub(crate) fn press_enter_to_continue() -> Result<()> {
     input(&format!("\n{}", "Press Enter to continue...".bright_blue()))?;
     Ok(())
 }
+
+pub(crate) fn ensure_gh_setup() -> Result<()> {
+    let output = Command::new("gh").arg("--version").output()?;
+    if !output.status.success() {
+        bail!("gh is not installed. Try installing with `brew install gh`")
+    }
+    let output = Command::new("gh").arg("auth").arg("status").output()?;
+    if !output.status.success() {
+        bail!("gh is not authenticated. Try running `gh auth login`")
+    }
+    let stderr = String::from_utf8(output.stderr)?;
+    if !stderr.contains("Logged in to github.com") {
+        bail!("gh is not logged in. Try running `gh auth login`")
+    }
+
+    println!("{}", "GitHub CLI is configured ✓".bright_green());
+
+    Ok(())
+}
+
+pub(crate) fn commit_all_into_branch(branch: &str) -> Result<()> {
+    let ic = ic_dir();
+
+    {
```

Comment:
NCR

Comment.

Seems simpler to just fail. My assumption here is that it is not likely that such a branch already exists. If I'm right, then trying to optimize the UX in that case is likely premature. IIUC, `git checkout -b $B` fails if B already exists.